Mod Menu Helper/assets/modmenu/lang/el_gr.json

```diff
@@ -0,0 +1,53 @@
+{
+  "modmenu.dropInfo.line1": "Οι τροποποιήσεις του FO έχουν  §9✎§r ή §8✎§r",
+  "modmenu.dropInfo.line2": "οι εναλλακτικές έχουν §c*§r, οι υπόλοιπες δεν υποστηρίζονται",
+  "modmenu.summaryTranslation.slight-gui-modifications": "Animations, έκδοση πακέτου στο μενού\n§9✎§8 Animations, χρώματα, εμφάνηση FPS, ...",
```

Comment:
Shouldn't "animations" be "κινούμενα σχέδια"?

Reply:
not the meaning that animation here means